mobile/lib/features/wallet/wallet_screen.dart

```diff
@@ -70,14 +70,15 @@ class _WalletScreenState extends State<WalletScreen> {
                           runSpacing: 10,
                           children: [
                             ValueDataRow(
-                                type: ValueType.amount,
-                                value: sendPaymentChangeNotifier
-                                    .pendingPayment?.decodedInvoice.amountSats,
+                                value: AmountText(
+                                    amount: (sendPaymentChangeNotifier
+                                        .pendingPayment
+                                        ?.decodedInvoice
+                                        .amountSats)!), // TODO: what to do when null
```

Comment:
Maybe we can omit the whole row if the value is null?

Reply:
right now the `ValueDataRow` will display the row, but show an empty space without the value. We could modify that behaviour, but it's a decent fallback (the layout doesn't change).

Reply:
> right now the ValueDataRow will display the row, but show an empty space without the value

really? to me it seems like the null value would be cast to an `Amount` which would cause an exception